Source/WebCore/platform/graphics/win/FontCacheWin.cpp

```diff
@@ -608,7 +608,7 @@ Vector<FontSelectionCapabilities> FontCache::getFontSelectionCapabilitiesInFamil
     LOGFONT logFont;
     logFont.lfCharSet = DEFAULT_CHARSET;
     StringView truncatedFamily = StringView(familyName).left(static_cast<unsigned>(LF_FACESIZE - 1));
-    truncatedFamily.getCharacters(ucharFrom(logFont.lfFaceName));
+    truncatedFamily.getCharacters(unsafeMakeSpan(ucharFrom(logFont.lfFaceName), truncatedFamily.length()));
```

Comment:
This unsafeMakeSpan length is kind of a tautology. Would be nicer to have a span() helper that took a LOGFONT as an argument. I guess it's not worth it in this patch. But I'm trying to convey how I think of "good" use of unsafeMakeSpan vs "bad".